SECURITY.md

```diff
@@ -0,0 +1,203 @@
+# Security Release Process
+
+Rook is a large growing community of storage providers, contributors, and users. The Rook community
+has adopted this security disclosure and response policy to ensure we responsibly handle critical
+issues.
+
+This policy is adapted from the policies of the following CNCF projects:
+
+* [Envoy](https://github.com/envoyproxy/envoy)
+* [CoreDNS](https://github.com/coredns/coredns)
+
+The Rook community credits and appreciates the example and security best practices that they have
+published openly.
+
+## Product Security Team (PST)
+
+Security vulnerabilities should be handled quickly and sometimes privately. The primary goal of this
+process is to reduce the total time users are vulnerable to publicly known exploits.
+
+The Product Security Team (PST) is responsible for organizing the entire response including internal
+communication and external disclosure.
+
+The initial Product Security Team will consist of the set of maintainers that volunteered.
+
+### Mailing lists
+
+* [security@rook.io](mailto:security@rook.io): for any security concerns. Received by Product
+  Security Team members, and used by this Team to discuss security issues and fixes.
+* rook-distributors-announce@lists.cncf.io: for early private information on Security patch
+  releases. See below how Rook distributors can apply for this list.
+
+## Disclosures
+
+### Private Disclosure Processes
+
+If you find a security vulnerability or any security related issues, please **DO NOT** file a public
+issue. **Do not** create a Github issue. Instead, send your report privately to
+[security@rook.io](mailto:security@rook.io). Security reports are greatly appreciated and we will
+publicly thank you for it.
+
+Please provide as much information as possible, so we can react quickly. For instance, that could
+include:
+
+* Description of the location and potential impact of the vulnerability
+* A detailed description of the steps required to reproduce the vulnerability (POC scripts,
+  screenshots, and logs are all helpful to us)
+* Whatever else you think we might need to identify the source of this vulnerability
```

Comment:
How about changing it to something like this `we might need to identify the source and possibly even fix of this vulnerability`
The idea behind it is that maybe the person has already found a fix for it as well.

Reply:
good idea @galexrt, i'll add more language to suggest that providing a fix would be desirable too